src/@carbon-info/constants/links.ts

```diff
@@ -7,7 +7,7 @@ export const Path = {
   Socials: {
     LinkedIn: "https://www.linkedin.com/company/switcheo/",
     Twitter: "https://twitter.com/0xcarbon",
-    Discussion: "https://t.me/carbon_ecosystem",
+    Discussion: "https://docs.carbon.network/module-architecture/carbon-evm",
```

Comment:
can you create another section for docs links? since we have links for both carbon-evm and carbon-core